dotnet-desktop-guide/net/winforms/compiler-messages/wfo1000.md

```diff
@@ -0,0 +1,59 @@
+---
+title: Compiler Error WFO1000
+description: Learn more about compiler error WFO1000. Property does not configure the code serialization for its property content.
+ms.date: 02/28/2025
+f1_keywords:
+  - "WFO1000"
+helpviewer_keywords:
+  - "WFO1000"
+---
+
+# Compiler Error WFO1000
+
+**Version introduced:** .NET 10 Preview 1
```

Comment:
No, this was introduced in NET9 RC1 https://learn.microsoft.com/en-us/dotnet/core/compatibility/windows-forms/9.0/security-analyzers